### ISSUE TITLE
Conceal is unset on color codes

### ISSUE THREAD

**Issue by bootmii** [rawles/teletext-editor#67]
I'm wondering if this behavior reflects real-world teletext decoders' behavior. I'm all for bug-for-bug compatibility with real systems that will be displaying edit.tf pages, especially because it subjects artists to the same quirks that will affect the display of their art, but if this bug doesn't reflect real behavior, then it needs to be removed.

**Comment by rawles** [comment]
Page 76 of ETS 300 706 (May 1997):

"Conceal ("Set-At") The following characters up to the end of the row, or until a Colour Code attribute (codes 0/0 to 0/7 or 1/0 to 1/7) is encountered, are to be displayed as SPACES until revealed by a decoder or user operation."

**Comment by pjfdirect** [comment]
Yep, looks like the editor is spec. I expect it's a feature, eliminating the need for a 'reveal' code. If I want to conceal something in between two chunks of visible teletext, dropping a graphic code after the stash opens the rest of the line up for visible art.

I did get a moment where it surprised me, when I'd drop my graphic code right after the 'conceal' code on the line. But I just trained myself to call my graphic code before the conceal, since it takes up no more space on the grid than the other (unworkable) way round.
